packages/react/navigation-menu/src/NavigationMenu.tsx

```diff
@@ -462,8 +464,8 @@ const NavigationMenuTrigger = React.forwardRef<
         </FocusGroupItem>
       </Collection.ItemSlot>
 
-      {/* Proxy tab order between trigger and content when viewport is in use */}
-      {open && context.viewport && (
+      {/* Proxy tab order between trigger and content */}
+      {open && (
```

Comment:
can you help me understand why the proxy is available for non-portalled content as well now?

Reply:
For re-enabling tabbables and entering the content. We could use a different method for non-viewport (which doesn't use the proxy element) but that seemed more complex than simply applying the same solution for both.